mmdt/mmdt/settings.py

```diff
@@ -12,24 +12,26 @@
 
 import os
 from pathlib import Path
-
 from decouple import config
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = 'django-insecure--ld-c!55j+l1gr@alfh&w#+flyd@r*y^wy_r+sokxv&i6d0%tg'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = config('DEBUG', cast=bool)
+ENVIRONMENT = os.getenv('ENVIRONMENT', 'development')
```

Comment:
If we write like this, you better mention how can we change the ENVIRONMENT in this case. Current codebase always return `development` when user didn't define  `os.getenv('ENVIRONMENT') . I will remove this code, just simply add DEBUG key in .env file it will solve it.

Reply:
```
    AWS_ACCESS_KEY_ID = 'your AWS S3 key'
    AWS_SECRET_ACCESS_KEY = 'your AWS S3 secret key '
    AWS_STORAGE_BUCKET_NAME = 'your S3 bucketname '
    DEBUG = True or False
```